File: fizzbuzz/javascript/src/Fizzbuzz.js
var fizzBuzz = function (chiffre) {
  var resultat = "";
  if (chiffre % 3 == 0){
    resultat += "fizz";
  }
  if (chiffre % 5 == 0){
    resultat += "buzz";
  }
  if (resultat == "") {
    resultat = chiffre;
  }
  return resultat;
};

File: fizzbuzz/haskell/FizzBuzz.hs
import Test.Hspec

data FBType = Same Integer | Fizz deriving (Eq, Show)

fizzBuzz :: Integer -> FBType
fizzBuzz x = x

main = hspec $ do
  describe "FizzBuzz should" $ do
    it "return 1" $
      (fizzBuzz 1) `shouldBe` 1
    it "return Fizz" $
      (fizzBuzz 3) `shouldBe` Fizz
